phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -1480,8 +1500,10 @@ public List<Mutation> getMutationList() {
                         GLOBAL_MUTATION_INDEX_COMMIT_FAILURE_COUNT.update(numFailedPhase3Mutations);
                     }
                 } finally {
-                    mutationCommitTime = EnvironmentEdgeManager.currentTimeMillis() - startTime;
-                    GLOBAL_MUTATION_COMMIT_TIME.update(mutationCommitTime);
+                    if (mutationCommitTime != 0) {
```

Comment:
Didn't understand this check

Reply:
@tkhurana this was leftover of previous commits. Cleaned it.